### PR TITLE
Season5/down in the river to pray

### PR DESCRIPTION
**Pull request by nocollier**
This work is ready for comment, I have checked each part and listened to several midi files. 

To make the merge a little easier I did not yet change the makefile, I figured we can change that in your PR.

Feel free to be picky, I tried to mostly stick with you style but it is my first attemp.

### CODE REVIEW

**Comment by Jim-Duke** [comment]
- Williard should be Willard.
- There is a separate "arranger" variable; so Brock Willard can be assigned to that
- I like using bar markers - they aren't technically needed; but they do help check to see if you misaligned things or put the wrong duration on a note.
- I like to force line breaks to make the systems on our output match the systems on the original music.  I find that makes it easier to do a visual inspection.
- The original music has the Tenor lines on the "treble 8" clef (one octave lower than the normal treble clef).  So all the tenor lines are an octave too high.  Easily fixed.  Just add a "\clef "treble_8"" line.  And start the music phrase with one less apostrophe.

Still listening and examining notes.  So far all looks correct.  But I'll post this comment before finishing that.

Comments on the arrangement
- It seems strange.  I suppose I could get used to it.  I wonder if Scott realizes it.
- They just put it down in the key of "C", even though the music is in the key of B.

File: season_5/down_in_the_river_to_pray.ly [hunk 1]
```diff
@@ -0,0 +1,729 @@
+\version "2.21.6"
+
+\include "../../LilypondTemplates/midi.ly"
+
+Title = "down_in_the_river_to_pray"
+
+\header {
+  title = "Down in the River to Pray"
+  composer = "George H. Allan, Brock Williard"
```
Comment (Jim-Duke) [inline season_5/down_in_the_river_to_pray.ly]: Williard should be Willard

File: season_5/down_in_the_river_to_pray.ly [hunk 2]
```diff
@@ -0,0 +1,729 @@
...
+  \tempo 2=55
+  \set Score.markFormatter = #format-mark-circle-numbers
+
+  % lead-in
+  r1
+  r1
+
+  % alto/tenor duet
+  r1 r1 r1 r1 r1 r1 r1 r1 r1
+  r1 r1 r1 r1 r1 r1 r1 r1 r1
```
Comment (Jim-Duke) [inline season_5/down_in_the_river_to_pray.ly:28]: This is optional; but I like putting bar checks in the music.  It catches lots of errors; at least it does for me (wrong duration, skipped a note, forgot a change in time signature, etc.).

Also, for one of the voices I like to add forced line breaks (and stop linebreaks) by adding "\noBreak", and "\break" respectively as needed to force our systems to match the systems in the original music.  I find that it's much easier to make a visual inspection of notes that way.

File: season_5/down_in_the_river_to_pray.ly [hunk 3]
```diff
@@ -0,0 +1,729 @@
...
+  
+  % page 8, system 2
+  b4) b8 b8 b8 b8 b4
+  r2 a2
+  gis1\fermata
+  
+}
+
+tenorOneMusic = \relative c''{
+  \key c \major
```
Comment (Jim-Duke) [inline season_5/down_in_the_river_to_pray.ly:344]: The music has the Tenors using a tenor cleff (a G clef with an 8 under it).  In lilypond you can specify that with "\clef "treble_8"".  As a result, of this mistake, the tenor notes are all one octave too high.  That's easily corrected by deleting one of the apostrophe's from the relative starting pitch.

**Review by Jim-Duke — CHANGES_REQUESTED** [review]

File: season_5/down_in_the_river_to_pray.ly [hunk 4]
```diff
@@ -0,0 +1,729 @@
...
+  e2. r4
+  e1~ e1~ e1~
+
+  % page 6, system 2
+  e1~ e1~ e1
+  r2 r4 e8 r8
+
+  % page 7, system 1
+  r1
+  a4 g4 fis4( fis8) b8
```
Comment (Jim-Duke) [inline season_5/down_in_the_river_to_pray.ly]: The 1/8th note B at the end of the measure before the 3/2 needs to go down.  The nearest B is up; so you need to add a comma.  And the F-sharp in the next measure will then need to be forced up with an apostrophe.  Same will hold for the Bass II part, of course.

File: season_5/down_in_the_river_to_pray.ly [hunk 5]
```diff
@@ -0,0 +1,729 @@
...
+  e4 c4 e2(
+  fis4) e4 d4 c4
+  \time 3/2
+  b1~ b4 r4\fermata
+
+  % page 7, system 2
+  \time 2/2 \tempo 2=60
+  r4 b'4 b4 b4
+  b4 a4 a4( gis4)
+  gis4 fis4 gis4~ gis8 gis8
```
Comment (Jim-Duke) [inline season_5/down_in_the_river_to_pray.ly]: Almost missed this.  The dashed slurs/ties - I don't think they're actually slurs or ties.  I think they mean something like don't take a breath or something.  In this case (Alto voice, Page 7, system 2, last measure) - look at the words.  You can't tie the g# quarter note with the g# eighth note - you need to articulate two different words with those notes.  If we tie them, the rhythm in the MIDI won't match what's needed.  I think for creating MIDI's we should omit all dashed slurs.  Most of the time it's a slur and the rhythms will come out in the MIDI.  But in several places it changes the rhythm.  I won't identify more of those - scan through the music to find them and remove them.

**Review by Jim-Duke — CHANGES_REQUESTED** [review]

**Comment by Jim-Duke** [comment]
Also, during this review I again realized how much I depend on measure numbers to review the music efficiently and make corrections.  And that includes noting the measure number in comments in the source file.  To make the measure numbers in the PDF we produce match the measure numbers in the original, you need to add the following line after the lead-in:

`\set Score.currentBarNumber = #1`

It only needs to be added to one voice.  I usually add all the breaks and other Score-wide things to the first voice in the file.

I now see that you made several commits.  I'll hold off on making my changes until after I see what you committed.

**Comment by nocollier** [comment]
This arrangement is weird. With that many consistent sharps, I knew it must have not really been in C. This is a trend I have seen in orchestral work as well--it is written in some key, but you don't get to know up front what it is.
 
I don't really care for the arrangement either, but as you say I could probably get used to it. From my view, this is a simple song that sounds best simple, but the arrangement likes to add additional moods and moving lines in strange places. We will see how it goes.

I made the changes you suggested before I saw your last comment, feel free to do what is easiest. If you like what you have we can even reject this PR and you can just push what you have. I will use all your suggestions from the beginning on the next one.

**Comment by Jim-Duke** [comment]
If needed I can make commits to your branch.  No reason to reject the PR.  The vast majority of the file is correct.  The hard part - the notes and rhythms - is correct; with a few exceptions that I noted.  I'll look over your changes and I think we can commit this.  If you could look over my PR we can push these forward.  I'll work on Ezekiel, since I have the sheet music.  You keep working on digital content.  There are two more scores available to work on.  I'll be getting the first shipment of sheet music this Friday - and I'll make scans so we can keep working things.

**Comment by nocollier** [comment]
Ok, I will try to get to review your score tonight or maybe tomorrow. Thanks!

**Review by Jim-Duke — APPROVED** [review]

**Pull request merged**